src/Worker.js

```diff
@@ -61,6 +63,7 @@ class Worker extends EventEmitter {
             this.emit('message', { type: jsonMessage.type, status: messageStatuses.proceed });
         } catch (error) {
             this.emit('error', error);
+            if (!this.options.ackOnMessageError) throw error;
```

Comment:
I think its not clear from the name of the variable that you are going to throw an error

Reply:
the error is for cancelling the handling and send the message to sqs again, so queue-router will handle this message again next time

Reply:
in this way i am "sending nack"